docs/usage/shoot_updates.md

```diff
@@ -0,0 +1,93 @@
+# Shoot Updates and Upgrades
+
+This document describes what happens during shoot updates (changes incorporated in a newly deployed Gardener version) and during shoot upgrades (changes for version controllable by end-users).
+
+## Updates
+
+Updates to all aspects of the shoot cluster happen when the gardenlet reconciles the `Shoot` resource.
+
+### When are Reconciliations Triggered
+
+Generally, when you change the specification of your `Shoot` the reconciliation will start immediately, potentially updating your cluster.
+
+You can also annotate your shoot with special operation annotations (see [this document](shoot_operations.md)) which will cause the reconciliation to start due to your actions.
+
+There is also an automatic reconciliation by Gardener.
+The period, i.e., how often it is performed, depends on the configuration of the Gardener administrators/operators.
+In some Gardener installations the operators might enable "reconciliation in maintenance time window only" ([more information](shoot_maintenance.md#cluster-reconciliation)) which will result in at least one reconciliation during the time configured in the `Shoot`'s `.spec.maintenance.timeWindow` field.
+
+Please note that you can also confine the reconciliation triggered due to your specification updates to the cluster's maintenance time window. Please find more information [here](shoot_maintenance.md#confine-specification-changesupdates-roll-out).
```

Comment:
Can you move this sentence up, to have it after related line 11 (`Generally, when you change the specification`)?